ccd-definition/AuthorisationCaseField/CareSupervision/magistrate.json

```diff
@@ -1213,7 +1241,7 @@
   {
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
-    "CaseFieldID": "courtBundleTabHeading",
```

Comment:
added the original field to AuthorisationCaseEvent/CareSupervision/DEPRECATED.json